basil/HL/mercury.py

```diff
@@ -28,11 +28,11 @@ def __init__(self, intf, conf):
 
     def init(self):
         super(Mercury, self).init()
-        self._addresses = []
-        for a in range(16):  # Check all possible addresses
-            self.write(bytearray.fromhex("01%d" % (a + 30)) + "TB".encode())  # Tell board address
-            if self.get_address(a):
-                self._addresses.append(a)
+        self._board_numbers = []
+        for b in range(16):  # Check all possible board numbers
+            self.write(bytearray.fromhex("01%d" % (b + 30)) + "TB".encode())  # Tell board number
+            if self.get_board_number(b):
+                self._board_numbers.append(b)
```

Comment:
Personally, I refrain from using single-character variables; maybe `board_n` or even `board_number` is a bit easier to read. Just wanted to point it out, this is no actual change request. Hit "resolve" if you want to keep it as it is.